doc/source/changes/version_0_28.rst.inc

```diff
@@ -1,7 +1,63 @@
-﻿New features
+﻿Backward incompatible changes
+-----------------------------
+
+* changed behavior of operators `session1 == session2` and `session1 != session2`: returns a boolean
+  (closes:issue:`516`):
+
+    >>> s1 = Session([('arr1', ndtest(2)), ('arr2', ndtest((2, 2)))])
+    >>> s2 = Session([('arr1', ndtest(2)), ('arr2', ndtest((2, 2)))])
+    >>> # equivalent to all(s1.equals(s2))
+    >>> s1 == s2
```

Comment:
this whole example is wrong/would not work.
after your change, s1 == s2 will return something like:
```python
>>> # should be almost equivalent to Session([(k, s1[k] == s2[k]) for k in s1.keys()])
>>> res = s1 == s2
>>> res.arr1
a    a0    a1
   True  True
>>> res.arr2
a\b    b0    b1
 a0  True  True
 a1  True  True
```